.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
```

Comment:
Do we want to add `end_of_line = lf`? I'd say yes.

Reply:
Agreed. Added.